animation_nodes/nodes/sound/sound_intensity.py

```diff
@@ -0,0 +1,63 @@
+import bpy
+import numpy
+from bpy.props import *
+from ... data_structures import DoubleList
+from ... base_types import AnimationNode, VectorizedSocket
+
+reductionFunctionItems = [
+    ("MEAN", "Mean", "Sample the frequency bins by computing the mean of frequency bins", "", 0),
+    ("MAX", "Max", "Sample the frequency bins by computing the maximum of frequency bins", "", 1)
+]
+reductionFunctions = {
+    "MEAN" : numpy.mean,
+    "MAX" : numpy.max
+}
+
+class SoundIntensityNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_SoundIntensityNode"
+    bl_label = "Sound Intensity"
+    errorHandlingType = "EXCEPTION"
+
+    reductionFunction: EnumProperty(name = "Reduction Function", default = "MAX",
+        description = "The function used to sample frequency bins", items = reductionFunctionItems)
+    smoothingSamples: IntProperty(name = "Smoothing Samples", default = 5, min = 0,
+        description = ("The number of frames computed to smooth the output."
+        " High value corresponds to more accurate results but with higher execution time"))
+
+    useFrameList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput("Sound", "Sound", "sound")
+        self.newInput(VectorizedSocket("Float", "useFrameList",
+            ("Frame", "frame"), ("Frames", "frame")))
```

Comment:
```suggestion
            ("Frame", "frame"), ("Frames", "frames")))
```

Reply:
Since we might remove this node anyway, I will leave this for now.